src/Og.php

```diff
@@ -59,4 +190,79 @@ public static function groupManager() {
     return \Drupal::service('og.group.manager');
   }
 
+  /**
+   * Invalidate cache.
+   *
+   * @param $gids
+   *   Array with group IDs that their cache should be invalidated.
+   */
+  public static function invalidateCache($gids = array()) {
+    // @todo We should not be using drupal_static() review and remove.
```

Comment:
This is not really the way things are going in D8. State is problematic enough at time, let alone shared static global state :)